backend/__tests__/integration/auth/schedule.spec.ts

```diff
@@ -7,26 +18,90 @@ describe('schedule.spec.ts - Schedule Controller', () => {
 
   describe('POST /', () => {
     // eslint-disable-next-line
-    it('should create schedule', () => {});
+    it('should create schedule when availability is AVAILABLE and startTime and endTime are provided', async () => {
+      const { availability, startTime, endTime }: ScheduleDTO =
+        await createSchedule(testCreateSchedule);
+
+      expect(availability).toEqual(testCreateSchedule.availability);
+      expect(startTime).toEqual(testCreateSchedule.startTime.toISOString());
+      expect(endTime).toEqual(testCreateSchedule.endTime.toISOString());
+    });
+
+    it('should not create schedule when availability is UNAVAILABLE', async () => {
```

Comment:
for this test, i think we need to rediscuss the implementation details about the availabilities.